README.md

```diff
@@ -1,3 +1,4 @@
 # infinity-stones
 Soul stone
-Space stone
\ No newline at end of file
+Space stone
```

Comment:
This  could be a potential bug. Please fix this before you merge this branch into master